internal/cmd/local/local_install.go

```diff
@@ -148,6 +170,10 @@ func (i *InstallCmd) Run(ctx context.Context, provider k8s.Provider, telClient t
 }
 
 func parseVolumeMounts(specs []string) ([]k8s.ExtraVolumeMount, error) {
+	if len(specs) == 0 {
+		return nil, nil
+	}
```

Comment:
This allows tests to pass easily with zero-value struct fields, because the code below uses `make` which doesn't return a nil slice when there are no volume mounts.